hamocc/mo_ocprod.F90

```diff
@@ -488,12 +517,39 @@ subroutine ocprod(kpie,kpje,kpke,kbnd,pdlxp,pdlyp,pddpo,omask,ptho,pi_ph,psao,pp
               phosy_NO3(i,j,k)   = (1.-nh4uptfrac)*phosy*rnit  ! kmol N/m3/dtb - NO3 uptake during PP growth
               remin_aerob(i,j,k) = (dtr+phosy)*rnit            ! kmol N/m3/dtb - Aerob remin to ammonium  (var. sources)
             endif
+            if (use_river2omip) then
+              ocetra(i,j,k,iphosph) = ocetra(i,j,k,iphosph) + tdoclc_rem+tdochc_rem
+              if (.not. use_extNcycle) then
+                ocetra(i,j,k,iano3)   = ocetra(i,j,k,iano3)   + tdoclc_rem*rnit_tdoclc             &
+                                      &                       + tdochc_rem*rnit_tdochc
+                ocetra(i,j,k,ialkali) = ocetra(i,j,k,ialkali) - (rnit_tdoclc+1.)*tdoclc_rem        &
+                                      &                       - (rnit_tdochc+1.)*tdochc_rem
```

Comment:
A bit picky, but reads nicer
```fortran
ocetra(i,j,k,ialkali) = ocetra(i,j,k,ialkali) - tdoclc_rem*(rnit_tdoclc+1.)        &
                      &                       - tdochc_rem*(rnit_tdochc+1.)
```
to align the `tdoc*_rem` with the lines above and below.